ascii_output_v3.py

```diff
@@ -0,0 +1,100 @@
+"""
+Turns csv output of tax calculator to ascii output with uniform columns
+and transposes data so columns are rows, rows are columns
+
+"""
+
+#records the data of one csv row to be later written into a uniform column
+#also returns number of rows in result file
+def rec_col(data, col_size, temp):
+    nrows = 0
+    while (data != []):
+        col_elem = data.pop()
+        #cuts a data string from the end if length is larger than col_size
+        if (len(col_elem) > col_size):
+            cut = col_size - len(col_elem)
+            col_elem = col_elem[:cut]
+        else:
+        #adds spaces to fill out data string if length is smaller than col_size
+            col_elem = col_elem.ljust(col_size)
+        temp.write(col_elem)
+        nrows += 1
+    return nrows
+
+def ascii_output():
+    csv_results = "results_puf.csv" #input csv file
```

Comment:
It seems odd that we have a hard coded filename here that reads a particular output file that has a hard coded filename in test.py. Why not have some kind of flag that indicates, when one does the work that produces `results_puf.csv` that you also want what happens in this file? That way you wouldn't have to execute `test.py` and then this script.